tests/test_utils.py

```diff
@@ -34,7 +34,7 @@ def test_change_rv_size():
     new_size = np.array([4, 3], dtype="int32")
     rv_newer = change_rv_size(rv, new_size=new_size, expand=False)
     assert rv_newer.ndim == 2
-    assert isinstance(rv_newer.owner.inputs[1], Constant)
+    assert isinstance(rv_newer.owner.inputs[1].owner.inputs[1], Constant)
```

Comment:
Related, the input for non-scalar sizes is now a SpecifyShape

Reply:
https://github.com/aesara-devs/aesara/pull/680 should fix this as well.